sentry_sdk/integrations/opentelemetry/sampler.py

```diff
@@ -54,6 +54,20 @@ def dropped_result(span_context, attributes, sample_rate=None):
     if sample_rate:
         trace_state = trace_state.add(TRACESTATE_SAMPLE_RATE_KEY, str(sample_rate))
 
+    is_root_span = not (span_context.is_valid and not span_context.is_remote)
```

Comment:
if `span_context` is not valid, why is it a root span?

Reply:
Its the parent span context, updated the naming so it is not confusing.